src/main/java/hudson/plugins/mercurial/MercurialSCMSource.java

```diff
@@ -71,7 +67,7 @@ public MercurialSCMSource(String id, String installation, String source, String
         this.browser = browser;
         this.clean = clean;
     }
-    
+
```

Comment:
BTW for the future please avoid gratuitous diff hunks—makes reviewing harder, increases risk of merge conflicts, etc.